subsys/modem/backends/Kconfig

```diff
@@ -51,3 +51,14 @@ config MODEM_BACKEND_UART_ASYNC_RECEIVE_IDLE_TIMEOUT_MS
 endif
 
 endif # MODEM_BACKEND_UART
+
+config MODEM_BACKEND_STAT
+	bool "Modem backend statistics"
```

Comment:
A little help message to explain what it does?

Reply:
Help message added